### PR TITLE
Add Aqualair Logo

### PR DESCRIPTION
**Pull request by aqualair**
#### Hi! What is your PR about?

- [x] Add images for your space
- [ ] Add custom domain for your space
- [ ] Add alias for migrate your space
- [ ] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: spaces/aqualair.eth/logo [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (gokaiorg) [inline spaces/aqualair.eth/logo:1]: Please delete this file.

**Review by gokaiorg — CHANGES_REQUESTED** [review]
I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.page/#/aqualair.eth

File: spaces/aqualair.eth/logo (hunk 1 above)
Comment (aqualair) [reply spaces/aqualair.eth/logo]: done

Comment (aqualair) [reply spaces/aqualair.eth/logo]: i have filled all required fields, please check it again

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**